portal/src/locale-data/en.json

```diff
@@ -1462,7 +1462,18 @@
   "LoginMethodConfigurationScreen.lockout.threshold.failedAttempts.description": "This is the maximum failed attempts to login before the account is locked.",
   "LoginMethodConfigurationScreen.lockout.threshold.resetAfter.title": "Reset failed attempts after (minutes)",
   "LoginMethodConfigurationScreen.lockout.threshold.resetAfter.description": "The interval to reset failed attempts after the last failure.",
-  "LoginMethodConfigurationScreen.lockout.threshold.overall.description": "A user account will be locked after {attempts} failed login attempts. The failure attempts will be reset after {resetIntervalMins} minutes of the last failure. {resetIntervalHours, plural, =0 {} other {(i.e. {resetIntervalHours} hours)}}",
+  "LoginMethodConfigurationScreen.lockout.threshold.overall.description": "A user account will be locked after {attempts} failed login attempts. The failure attempts will be reset after {resetIntervalMins} minutes of the last failure. {resetIntervalHours, select, none {} other {(i.e. {resetIntervalHours} hours)}}",
+  "LoginMethodConfigurationScreen.lockout.duration.title": "Lockout duration",
+  "LoginMethodConfigurationScreen.lockout.duration.description": "Configure the number of minutes an account will remain locked when it has exceeded the lockout threshold.",
+  "LoginMethodConfigurationScreen.lockout.duration.duration.title": "Lockout Duration (minutes)",
+  "LoginMethodConfigurationScreen.lockout.duration.duration.description": "This determines how long the account will remain locked.",
+  "LoginMethodConfigurationScreen.lockout.duration.backoff.title": "Backoff Factor",
+  "LoginMethodConfigurationScreen.lockout.duration.backoff.description": "This value will increase the lockout duration exponentially.",
+  "LoginMethodConfigurationScreen.lockout.duration.max.title": "Maximum Lockout Duration (minutes)",
+  "LoginMethodConfigurationScreen.lockout.duration.max.description": "This will allow you to put a cap on the lockout duration.",
+  "LoginMethodConfigurationScreen.lockout.duration.overall.description.noBackoff": "If a user account has exceeded the lockout threshold, it will be locked for {durationMins} minutes. For every subsequent failed attempt, the user must wait for another {durationMins} minutes until a correct password is entered.",
```

Comment:
Use `plural` for `durationMins` here.

Reply:
@louischan-oursky Why we need plural here?

Reply:
When it is 1, it shows `1 minutes` which is not grammatically correct.

Reply:
Oh right. Let me add `plural` for all messages if needed.